test/test_check_multiqc.py

```diff
@@ -0,0 +1,53 @@
+"""
+Test Hail Query functions.
+"""
+import shutil
+import pytest
+import toml
+from cpg_utils import to_path, Path
+from cpg_utils.config import set_config_paths, update_dict
+from cpg_utils.workflows.utils import timestamp
+
+from jobs.python_scripts import check_multiqc
+
+
+DEFAULT_CONF = """
+[workflow]
+sequencing_type = 'genome'
+
+[qc_thresholds.genome.min]
+"MEDIAN_COVERAGE" = 10
+"PCT_PF_READS_ALIGNED" = 80
+[qc_thresholds.genome.max]
+"FREEMIX" = 0.04
+"PERCENT_DUPLICATION" = 25
+"""
+
+
+@pytest.fixture()
+def tmp_dir() -> Path:
+    dir_path = to_path('results') / timestamp()
+    dir_path.mkdir(parents=True, exist_ok=True)
+    yield dir_path
+    shutil.rmtree(dir_path)
```

Comment:
isn't this just replicating pytest's `tmpdir` fixture?

Reply:
Oh, didn't know about it! Thanks a lot @MattWellie  :D

Reply:
It's my favourite default fixture ❤️

Reply:
The only problem is that Hail's local backend doesn't work well with `/tmp`

Reply:
@MattWellie while you are here, would you please click "approve"?

Reply:
sample-metadata was removed from the requirements, which is killing the test runs.. should that be added back in?

Reply:
Oh, it actually should be `cpg-utils[workflows]` that installs `sample-metadata` now

Reply:
`cpg-utils` needs to be bumped :) https://github.com/populationgenomics/cpg-utils/pull/76

Reply:
Alright, tests are passing now, and the standard `tmpdir` fixture works here! Thanks @MattWellie :)